docs/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!--
 
-    Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2022, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
see line ~153

Reply:
Done

Reply:
nope :-) - I've said line ~153- the one with: `<group title="Implementation-specific Packages" packages="com.sun.activation*"/>`